libvast/src/system/application.cpp

```diff
@@ -64,6 +65,15 @@ auto make_root_command(std::string_view path) {
                                       "definitions"));
 }
 
+auto make_count_command() {
+  return std::make_unique<command>(
+    "count", "count hits for a query without exporting data", "",
+    opts("?count")
+      .add<bool>("node,N", "spawn a node instead of connecting to one")
```

Comment:
The whole `-N` thing seems to be an orthogonal concern. I wonder if it can somehow be factored. @tobim, any ideas?

Reply:
As a side note—currently, the customization point is not executed for `count -N`. We decided to move the customization point out of the start command some time in the future.

Reply:
We could move the `-N;--node` option to the root command, and check for it in `spawn_or_connect_to_node()`. That would simplify the commands and that function.

Reply:
I'll get this done in another PR once this is merged.